DataConnectors/CiscoFirepower/CiscoFirepowerEStreamerCollector.json

```diff
@@ -86,7 +85,7 @@
                                     "PrimaryKey"
                                 ],
                                 "label": "Run the following command to install and apply the CEF collector:",
-                                "value": "sudo wget -O cef_installer.py https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
+                                "value": "sudo wget https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/CEF/cef_installer.py&&sudo python cef_installer.py {0} {1}"
```

Comment:
-O cef_installer.py is missing. Please refer the sample template https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_CEF_template.json

Thanks,
Aditi Maheshwari